src/locales/en_US.json

```diff
@@ -22,5 +29,81 @@
     "basepack": {
       "run_new": "After uploading the above files to your Web App tap Run New Test to test your PWA again!"
     }
+  },
+  "text": {
+    "manifest_options": {
+      "top_section": {
+        "h1": "Manifest"
+      },
+      "summary_body": {
+        "h1": "Summary",
+        "p": "Easily update and upgrade your Web Manifest with our built-in Web Manifest editor"
+      },
+      "info": {
+        "h1": "Info"
+      },
+      "images": {
+        "h1": "Images",
+        "icons": {
+          "h3": "Upload App Icons"
+        },
+        "screenshots": {
+          "h3": "Generate Screenshots",
+          "p": "Specify the URLs to generate desktop and mobile screenshots from. You may add up to 8 screenshots or Store Listings."
+        }
+      },
+      "settings": {
+        "h1": "Settings",
+        "background_color": {
+          "h3": "Background Color"
+        },
+        "theme_color": {
+          "h3": "Theme Color"
+        }
+      },
+      "view_code": {
+        "h1": "View Code"
+      },
+      "titles": {
```

Comment:
not super happy with this delimitation. let me know if you guys would rather have it by entry type, meaning more like my objects that control what to show in the module.
so:
```json
{
  "name": {
    title: "",
    "description": "",
    "tooltip": "",
  }
}
```

Reply:
Hmmmmm, I think I actually prefer how you have it done here in the PR (: